src/fmgc/src/flightplanning/FlightPlanManager.ts

```diff
@@ -24,13 +24,17 @@
  */
 
 import { NXDataStore } from '@shared/persistence';
-import { WaypointConstraintType } from '@fmgc/types/fstypes/FSEnums';
```

Comment:
Can we please remove this enum from the FS typedefs? It never belonged there as it is one of ours. 😬

Reply:
done